src/main/kotlin/apply/application/MissionService.kt

```diff
@@ -82,6 +101,15 @@ class MissionService(
     fun getDataById(id: Long): MissionData {
         val mission = missionRepository.getById(id)
         val evaluation = evaluationRepository.getById(mission.evaluationId)
-        return MissionData(mission, evaluation)
+        val judgeItem = judgeItemRepository.findByMissionId(mission.id)
+        val evaluationItemOptional = evaluationItemRepository.findById(judgeItem?.evaluationItemId ?: 0L)
+        if (evaluationItemOptional.isPresent) {
+            return MissionData(
+                mission,
+                evaluation,
+                JudgeItemData(judgeItem, EvaluationItemData(evaluationItemOptional.get()))
+            )
+        }
+        return MissionData(mission, evaluation, JudgeItemData(judgeItem, null))
```

Comment:
c: `isPresent()` - `get()`으로 optional을 처리하는 것보단 `orElse`를 활용해보는 건 어떤가요?
아래 방법도 괜찮을 것 같아요
```suggestion
    fun getDataById(id: Long): MissionData {
        val mission = missionRepository.getById(id)
        val evaluation = evaluationRepository.getById(mission.evaluationId)
        val judgeItem = judgeItemRepository.findByMissionId(mission.id)
        val evaluationItem = getEvaluationItem(judgeItem)
        return MissionData(
            mission,
            evaluation,
            JudgeItemData(judgeItem, EvaluationItemData(evaluationItem))
        )
    }

    private fun getEvaluationItem(judgeItem: JudgeItem?): EvaluationItem {
        val evaluationItemOptional = evaluationItemRepository.findById(judgeItem?.evaluationItemId ?: 0L)
        return evaluationItemOptional.orElse(null)
    }
```

Reply:
메소드 분리하겠습니다~